sdk/storage/azure-storage-common/CHANGELOG.md

```diff
@@ -1,15 +1,23 @@
 # Release History
 
-## 12.1.0-beta.1 (Unreleased)
+## 12.1.0 (2021-08-10)
 
 ### Features Added
 
 ### Breaking Changes
 
 ### Bugs Fixed
 
+- Avoid time domain casting exception during request cancellation. (A community contribution, courtesy of _[johnwheffner](https://github.com/johnwheffner)_)
+
 ### Other Changes
 
+### Acknowledgments
+
+Thank you to our developer community members who helped to make Azure Storage better with their contributions to this release:
+
+- John Heffner _([GitHub](https://github.com/johnwheffner))_
```

Comment:
Thank you @johnwheffner! Your fix is now in the release - https://github.com/Azure/azure-sdk-for-cpp/releases/tag/azure-storage-common_12.1.0